jwst/stpipe/core.py

```diff
@@ -19,13 +19,21 @@
 
 class JwstStep(Step):
 
+    datamodel_access_times = 0
     spec = """
     output_ext = string(default='.fits')  # Output file type
+    modify_input = boolean(default=False)  # if True modify the input datamodel in-place, don't make a copy
```

Comment:
If we check if the step is part of a pipeline (has a `Step.parent`) and use that to determine if the input can be modified this addition to the spec would be unecessary.

Reply:
I tried this and got it working so I removed the new Step attribute.